protocols/minigames/chickenshock/2022-08-08-protocol.md

```diff
@@ -0,0 +1,30 @@
+# Meeting Transcript 08.08.2022
+
+What do we still want to add to chickenshock / what would be nice to have, what is left to do this sprint?
+
+- present: Martin, Levi
+- absent excused: -
+- absent unexcused: -
+
+## What do we want to add?
+
+- Pause menu with options menu and quit button.
+- If the player pauses he loses 1 point per question to prevent abuse of pausing
+- Should the pause menu stop the timer? Not 100% decided, but probably no
+- Option to change mouse sensitivity
```

Comment:
Not your job?
That's the job of the OS?

Reply:
What? If it was then every single shooter on the market wouldn't need a mouse sens option

Reply:
Okay, if you want to add it, then go for it.
But I would think that the user should have configured his system to his liking...

Reply:
I think there's a misunderstanding here, we don't project the crosshair on you cursor, instead we check for the cursor movement and use that to move the camera. This means we have a (currently static) multiplier to you mouse movement which we want to make variable so you have to move the mouse more or less to move the camera more or less. If that makes sense